stdlib/pstats.pyi

```diff
@@ -77,3 +77,16 @@ class Stats:
     def print_call_line(self, name_size: int, source: str, call_dict: dict[str, Any], arrow: str = "->") -> None: ...
     def print_title(self) -> None: ...
     def print_line(self, func: str) -> None: ...
+
+_Func: TypeAlias = tuple[str, int, str]
+
+def func_strip_path(func_name: _Func) -> _Func: ...
+def func_get_function_name(func: _Func) -> str: ...
+def func_std_string(func_name: _Func) -> str: ...
+
+_Callers: TypeAlias = dict[_Func, tuple[int, int, int, int]]
+_StatVal: TypeAlias = tuple[int, int, int, int, _Callers]
+
+def add_func_stats(target: _StatVal, source: _StatVal) -> _StatVal: ...
+def add_callers(target: _Callers, source: _Callers) -> _Callers: ...
+def count_calls(callers: _Callers) -> int: ...
```

Comment:
This doesn't look correct to me -- the values of the `callers` dictionary have to be objects such that adding 0 to one of the values won't raise an error: https://github.com/python/cpython/blob/bb578a0c304dffe43bb28b36b2b1c9153c78b659/Lib/pstats.py#L593-L598